ViewGenerator/tools/webpack/Plugins/CommonConfiguration.ts

```diff
@@ -121,7 +121,7 @@ let getCommonConfiguration = (libraryName: string, useCache: boolean, assemblyNa
                     }
                     return OutputDirectoryDefault + CssChunkPlaceholder;
                 },
-                chunkFilename: OutputDirectoryDefault + CssChunkPlaceholder
+                chunkFilename: OutputDirectoryDefault + "[id].css"
```

Comment:
We can put the placeholder in Resources.ts and it should be conditional to compiling in hot reload or not